requirements.txt

```diff
@@ -6,7 +6,6 @@ periodictable~=1.5
 picmistandard==0.0.16
```

Comment:
The PR that introduces the 1D Cartesian grid was merged in `picmistandard`.
@ax3l Do I understand correctly that, for the tests to pass here, we should do a new release of `picmistandard` (0.0.17) and update the above requirement?

Reply:
```suggestion
picmistandard==0.0.17
```